pkgs/development/python-modules/dlx/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+}:
+
+buildPythonPackage rec {
+  pname = "dlx";
+  version = "1.0.4";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "ef75bc9d590216ebde7d4811f9ae6b2d6c6dc2a54772d94ae13384dc517a5aae";
+  };
+
+  # This package contains no tests
+  doCheck = false;
```

Comment:
would prefer checking out from github if available. Unit tests are crucial in assuring your package isn't' broken by bumping dependencies